std_msgs/Header header
float64 latitude
float64 longitude
float64 hAcc
uint8 fixtype
uint16 yaw
uint16 pitch
uint16 roll
uint16 control_mode

